site/en/docs/android/custom-tabs/guide-ui-customization/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: "layouts/doc-post.njk"
+title: Customizing the UI
+seoTitle: "Guide: customizing the Custom Tab UI"
+date: 2023-04-21
+description: How to make a Custom Tab match the look and feel of your app.
+authors:
+  - sebastianbenz
+---
+
+One advantage of Custom Tabs is that it can be seamlessly integrated into your app. In this part of the Custom Tabs guide, you will learn how to change the appearance and behavior of a Custom Tab to match your app.
+
+## Set the color of the toolbar
+
+{% Columns %}
+
+{% Column %}
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/OEyGhy3ea1pZfqOYVFcA.png", alt="Custom Tab with a custom light color scheme", width="300", height="577", class="screenshot screenshot--filled" %}
+Light mode
+{% endColumn %}
+
+{% Column %}
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/1K0VdsxhR76P5UXBNc2I.png", alt="Custom Tab with a custom dark color scheme", width="300", height="577", class="screenshot screenshot--filled" %}
+Dark mode
+{% endColumn %}
+
+{% endColumns %}
+
+First, customizing the Custom Tab’s address bar to be consistent with your app's theme. The snippet below changes the default toolbar color by setting [setDefaultColorSchemeParams](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder?cmdf=setDefaultColorSchemeParams%20custom%20tabs#setDefaultColorSchemeParams(androidx.browser.customtabs.CustomTabColorSchemeParams)). If your app also supports a dark color scheme, set it via [.setColorSchemeParams(CustomTabsIntent.COLOR_SCHEME_DARK, …)](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setColorSchemeParams(int,androidx.browser.customtabs.CustomTabColorSchemeParams)).
+
+```java
+// get the current toolbar background color (this might work differently in your app)
```

Comment:
> (this might work differently in your app)

While I absolutely appreciate this as someone who struggles to read java, this feels unusual to call out, given the lack of similar messages throughout other code samples.